Features/AdminComponentPreview/README.md

```diff
@@ -0,0 +1,22 @@
+# Admin Component Preview (Flynt Feature)
+
+Add layout screenshots of a component to its component folder to show a preview in the front end and WordPress back end.
+
+Components can then be previewed in multiple scenarios:
+
+## Front end
+
+When logged in, click the 'Component Preview' button on the WordPress admin bar to overlay a preview image (with reduced opacity). For easy pixel perfect development, drag the images around or move them with the arrow keys.
+
+## Back end
+
+The preview image will be shown beside the component name in the flexible content group layout.
+
+Rollover a component name inside the "Add Component" dropdown menu to see the desktop and mobile preview images.
+
+## Usage
+
+Prepare a file size optimized  `preview-desktop.jpg` and `preview-mobile.jpg` for your component.
+
+- `preview-desktop.jpg` should be 1440x900 px.
+- `preview-mobile.jpg` should be 320x480 px.
```

Comment:
not sure if it's obvious, but should we mention that these files need to be in the components folder?